views/home/index.hbs

```diff
@@ -9,4 +9,29 @@
     <button class="signup-button" type="button" onclick="location.href = '/sessions/new';">Login</button>
   {{/if}}
   </main>
-</div>
+</div> --}}
+
+<div class="hero-image" style="height: 100%;">
+  <div class="hero-text">
+    <h1>Welcome to {{title}}</h1>
```

Comment:
please, add `class="title"` here, or change the test

Reply:
the dot in 'title' shouldn't be there, that's checking the page title, I'll fix that